### ISSUE TITLE
a11y red-flags: live regions

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#635]
**Description**
Since MOVE is a single-page application where content is dynamically loaded into regions of an already-loaded page, we need to take extra care to ensure ARIA live regions are marked appropriately.

This is especially important for areas containing dynamic notifications.

**Acceptance Criteria**
- [ ] figure out `aria-live` use for location search bar;
- [ ] announce dialogs / modals via `aria-live`;
- [ ] announce snackbars via `aria-live`.

**Additional Notes**
See [MDN Aria Live Regions docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) for more details, and [Accessible App](https://accessible-app.com/pattern/vue/notifications) for SPA-specific information.

**Comment by candu** [comment]
Removing this from beta milestone for now, as [NVDA doesn't always respect `aria-live`](https://github.com/nvaccess/nvda/issues/7906) - and so we have no reliable way to test it.

When we get a11y red-flags test results back from the Digital Accessibility Team, we can either resurrect / continue this or WONTFIX it (depending on said results).

**Comment by candu** [comment]
Closing this for now - we'll open more specific tasks once a11y prioritization is done.
